### DISCUSSION TITLE
Adding auth header to stream

### DISCUSSION THREAD

**Discussion by meganvw** [mpv-player/mpv#15009] [Q&A] [answered]
Hi, I use mpv to play audio stream URLs. I would like to stream a paid service, like soundcloud, which gives me access to premium content via an auth token that is added as a header to the request when playing the stream:

```
# request a track you want to stream
$ curl -X GET "https://api.soundcloud.com/tracks/TRACK_ID" \
       -H  "accept: application/json; charset=utf-8" \
       -H "Authorization: OAuth ACCESS_TOKEN"
```

Is there currently a way to add headers to the mpv stream request? I don't see a way, but wondering if I've missed it.

Thanks!

**Comment by guidocella** [comment, marked as answer]
`--http-header-fields`

**Reply by meganvw** [reply]
thank you!! I see it now in docs...
